src/RG_SIMO_1.py

```diff
@@ -318,28 +316,6 @@ def run(self, container, Inputs):
       csvfile.close()
 
     y_hist = np.array(y_hist).T
-    # print(v_hist)
-
-    fig, axs = plt.subplots(3)
```

Comment:
I had to remove this since it goes in conflict with the RAVEN plotting system causing a Crash.

Reply:
I am really sorry Andrea, I should have deleted the "FARM/src/RefGov_SIMO.py" and "FARM/src/RG_SIMO_1.py", as well as 2 folders and 2 xmls in the FARM/tests directory since they are the old version. I accidentally added them to the push...

- FARM/tests/RefGov_hard_coded_Test
- FARM/tests/RefGov_xmlABC_Test
- FARM/tests/test_RefGov_hard_coded.xml
- FARM/tests/test_RefGov_xmlABC.xml

Could you delete these 6 guys for me?

Reply:
yes absolutely.... i can do it....

Reply:
you want me to delete also the files in "src"?  src/RefGov_SIMO.py and src/RG_SIMO_1.py?

Reply:
Is it okay?

Reply:
Yes, please delete the following two files due to the old version

- FARM/src/RefGov_SIMO.py
- FARM/src/RG_SIMO_1.py

Thanks, Haoyu

Reply:
(if you are good with these modifications, you can approve (actually you already did) and merge the Pull Request, so I can officially add the submodule as a RAVEN official plugin)